_posts/2020-01-29-Introduction_to_the_IAC_Team.md

```diff
@@ -0,0 +1,121 @@
+Through this feedback we're trying to ensure that the user experience is as close to the requirements of the people using and contributing to the product.
+
+One valuable feedback vector is through the weekly [Office Hours](https://puppet.com/community/office-hours/) sessions.
+Once a week, members from the IAC team will down tools and be available to any queries coming in via Slack or the video conferencing session from Puppet Community contributors.
+If you need help with module development or testing, or you want to determine the best way to do something with a module on the Forge, then this is the place to get that help and send that feedback.
+Quite often, if a limitation or issue in a module has been highlighted here, we will bring this up in our next stand-up meeting, raise a JIRA ticket and ensure that it gets prioritisation for resources.
+There's no better feedback than the people using actively using the module ecosystem and we like to make the most of this valuable resource.
+ 
+ 
+### Types of Work
+Our number one remit and focus is to ensure that those who want to contribute to the Forge have as frictionless an experience as possible.
+That spans everything from a module's lifespan, through initial design / prototyping, to version 1.0.0, right up to the point where it's a veteran module with 100,000s of downloads and a wide and active contributor base.
+Our goal is to develop the tools and processes to ensure that is possible.
+One very important aspect of our job is the Content Triage rotation.
+Two members of the IAC team will take it in turns to review community contributors to Puppet Forge modules.
+We're there to ensure that the change meets the quality standard required but more important, we're there to help contributors understand how to do that.
+We're very fortunate to receive lots of high quality contributions or those willing to act upon feedback.
+The goal is to ensure that the quality of the Puppet modules remains high, whilst constantly being enhanced and reactive to the needs of the community.
+ 
+ 
+### Tech Stack
+Some folks may be curious as to what we use day-to-day to get the job done.
+There is quite a large amount of software, platforms and tools we use, some you'll likely have heard of and some internal proprietary tools.
+For internal tooling, we're in a constant effort to evolve them and make them available for use by the Puppet Community.
+
+### Development and Debugging
+For the most part, the IDE of choice for us is VSCode.
+It has support for Puppet manifests via [this extension](https://marketplace.visualstudio.com/items?itemName=jpogran.puppet-vscode).
+Some of the team also use RubyMine and Sublime Text can also be found on a lot of laptops.
+The [Ruby gem pry](https://rubygems.org/gems/pry/) is a heavily utilised one.
+This has proven an invaluable tool for debugging during development or investigation when reproducing issues reported to us.
```

Comment:
Consider moving the PDK and pdksync sections up here to live under the Development and Debugging header?

I think everything after tech stack should actually have the header bumped one level as they're subsections of that?

Reply:
Done, but that H5 level text is super small! Let me know what you think

Reply:
Ahhhhhhhhhhhh

What if we made the tech stack an H2? It feels like a new category from `Workflow and Processes` maybe?